engine/engine.go

```diff
@@ -136,6 +139,7 @@ func Start(ctx context.Context, startOpts Config, fn StartCallback) error {
 		startOpts.CloudURLCallback(cloudURL)
 	}
 
+	loader.Task("detecting platform").Complete()
```

Comment:
Thanks for your comment!
I think I'll refactor to use the first option since these tasks shall not take too much time!

Reply:
@vito With the second method, logs are not displayed :(
```
┣─┼─╮                                                                           
│ │ ▼ start engine                                                              
│ │ █ [0.21s] loading                                                           
│ │ ┻          
```

Reply:
@TomChv What you're seeing is probably because log output is hidden for completed vertices. If something is stuck or taking a while you'll see the logs, which seems to be the main goal of #4820. Sub-tasks will show up even after completion, but I think that might just be added noise, since most of the time you don't really care about the engine bootstrapping process.